spec/kvm.go

```diff
@@ -5,6 +5,7 @@ import "github.com/giantswarm/kvmtpr/spec/kvm"
 type KVM struct {
 	EndpointUpdater kvm.EndpointUpdater `json:"endpointUpdater" yaml:"endpointUpdater"`
 	K8sKVM          kvm.K8sKVM          `json:"k8sKVM" yaml:"k8sKVM"`
+	NodeController  kvm.NodeController  `json:"nodeController" yaml:"nodeController"`
```

Comment:
I am so bad at this, I had to look it up because I was not sure. The internet says N comes after M in the alphabet. :D

Reply:
What? E - K - N all good, isn't it?

Reply:
But it is M - N and not N - M. :D

Reply:
hehe, i opened whole file now. I see it i see it, i was blind.

;)